src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Figure implements Area, Output{
```

Comment:
Don't begin class or method implementation with an empty line. Remove the redundant empty line at the beginning of the class definition.